.github/workflows/run-tests.yml

```diff
@@ -53,8 +53,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, '3.10', 3.11]
-        cdk-version: [cdk, cdkv2]
+        python-version: [3.7, 3.8, 3.9, '3.10', 3.11, 3.12]
```

Comment:
Just curious, any reason to still support Python 3.7 since it's EOL? https://devguide.python.org/versions/

Reply:
I have a branch that removes 3.7 support, but we're following the python support policy for AWS SDKs and Tools (https://aws.amazon.com/blogs/developer/python-support-policy-updates-for-aws-sdks-and-tools/), which would drop 3.7 support at the end of this month.

I was hoping to get 3.12 support released quickly and then drop support for 3.7 next month, but depending on how long it takes to work out the remaining 3.12 issues, we might end having 3.12 support + dropping 3.7 in the same release if it ends up getting pushed to next month.

Reply:
@jamesls It's the new year, any chance you could take a look at this again?